apps/arena/lib/arena/entities.ex

```diff
@@ -64,7 +64,8 @@ defmodule Arena.Entities do
         on_bush: false,
         bounties: [],
         selected_bounty: nil,
-        bounty_completed: false
+        bounty_completed: false,
+        current_basic_animation: 0
```

Comment:
Isn't this value supposed to be 1? to replicate what `get_skill_animation` is doing